Bottom: 62b39660db73dc8788d632994d7497faef7b4dca
Top:    24715b4db27963bdcdb321576847c4da74da5c02
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-28 21:28:08 +0100

parse_y.y: Don't bother setting LayerFlag[1,2] for elements - it isn't used


---

diff --git a/src/parse_y.y b/src/parse_y.y
index 0bbbfaf..fcf57cc 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -232,8 +232,6 @@ parsepcb
 		    if (yyPCB != NULL)
 		      {
 			/* This is the case where we load a footprint with file->open, or from the command line */
-			LayerFlag[0] = true;
-			LayerFlag[1] = true;
 			yyData->LayerN = 2;
 			CreateNewPCBPost (yyPCB, 0);
 			ParseGroupString("1,c:2,s", &yyPCB->LayerGroups, yyData->LayerN);
